src/reporter.js

```diff
@@ -126,9 +137,15 @@ module.exports = async (answers) => {
                         .then((res) => {
                           result.server = url;
                           result.report = res.data;
+
                           resolve(result);
                         })
                         .catch((err) => {
+                          console.log("");
```

Comment:
What is the purpose of the empty string?

Reply:
To ensure there is a new line in the console because the spinner class used does not count as a line.